Add deploy in Heroku mini tutorial and note in pipenv section (#69)
rootstrap/tech-guides: python/cookiecutter-django.md

  - `requirements.txt` from the root folder of the project.
  - `requirements/base.txt`, `requirements/local.txt`, `requirements/production.txt` and the `requirements` folder.

NOTES:
- Remember, when you install a new dependency, check if it will be needed at production or not. In the case that it won't be needed, add the flag `--dev` at the end of the installation command.

> - In the Pipfile, try to avoid the use of `"*"` for the library versions. For each library set the used version in the project. This is to avoid future compatibility errors.

Reviewer: ```suggestion
- In the Pipfile, try to avoid the use of `"*"` for the library versions. Set a specific version for each library in order to avoid future compatibility errors.
```
